packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1734,7 +1734,13 @@ export class ContainerRuntime
 					subRequest.url.startsWith("/"),
 					0x126 /* "Expected createSubRequest url to include a leading slash" */,
 				);
-				return dataStore.request(subRequest);
+				return subRequest.url === "/"
```

Comment:
not sure where we are at yet, but eventually we'll need to add resolve handle to the data store object, so it can resolve sub-objects like dds. i also wonder if this case would be better handled there. the runtime they becomes mainly about routing to the correct object type, and not full fulling the resolution. i think that will make it easier for consumers that do use the request pattern with mixins, as it will keep there future resolve handle mixing limited to the datastore layer